src/main/resources/dbreplica/migration/V1.0.7__configure_replication_plsql.sql

```diff
@@ -67,7 +61,7 @@ DROP TRIGGER IF EXISTS trg_on_task_upsert ON cft_task_db.tasks;
 -- Add the task upsert trigger.
 --
 CREATE TRIGGER trg_on_task_upsert before insert or update on cft_task_db.tasks
-  for each row execute function on_task_upsert();
+  for each row when (NEW.case_id is not null) execute function on_task_upsert();
```

Comment:
Does the cft_task_db have a not null constraint on the `case_id`.  I think that's how it was designed but just to check it's there if we are going rely on it we need the guarantee.

Reply:
cft_task_db does not have a not null constraint on the case_id

Reply:
👍